js_modules/dagit/packages/app/README.md

```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
```

Comment:
I wonder if we should rm this file? I guess it's useful but I wonder if there was another readme / if we should encourage people to use the contributing docs?

Reply:
Yeah I think it makes sense to delete it for now and replace it with something more specific to this app.